src/node_snapshotable.cc

```diff
@@ -1416,6 +1416,7 @@ StartupData SerializeNodeContextInternalFields(Local<Object> holder,
   if (index == BaseObject::kEmbedderType) {
     int size = sizeof(EmbedderTypeInfo);
     char* data = new char[size];
```

Comment:
```suggestion
    char* data = new char[size]{};
```

or 

```C++
    char* data = new char[size]();
```

... and then you do not need the `memset`. 

See https://stackoverflow.com/questions/2204176/how-to-initialise-memory-with-new-operator-in-c


(Your code is fine and it is *not* request for change.)

Reply:
TIL, thanks! Updated to use the parentheses and added a comment in case it is confusing for future readers.